#goal definition
string goal_obj
string bin_content
---
#result definition
string objID
geometry_msgs/Twist objPose
---
#feedback
string status
